src/uu/cksum/src/cksum.rs

```diff
@@ -448,15 +458,42 @@ pub fn uu_app() -> Command {
                 .help("emit a raw binary digest, not hexadecimal")
                 .action(ArgAction::SetTrue),
         )
+        /*.arg(
+            Arg::new(options::STRICT)
+                .long(options::STRICT)
+                .help("exit non-zero for improperly formatted checksum lines")
+                .action(ArgAction::SetTrue),
+        )*/
+        .arg(
+            Arg::new(options::CHECK)
+                .short('c')
+                .long(options::CHECK)
+                .help("read hashsums from the FILEs and check them")
+                .action(ArgAction::SetTrue)
+                .conflicts_with("tag"),
+        )
         .arg(
             Arg::new(options::BASE64)
                 .long(options::BASE64)
-                .short('b')
                 .help("emit a base64 digest, not hexadecimal")
                 .action(ArgAction::SetTrue)
                 // Even though this could easily just override an earlier '--raw',
                 // GNU cksum does not permit these flags to be combined:
                 .conflicts_with(options::RAW),
         )
+        .arg(
+            Arg::new(options::TEXT)
+                .long(options::TEXT)
+                .short('t')
+                .hide(true) // for legacy compatibility, no action
```

Comment:
GNU 9.5 does the following:

```console
$ ../gnu/src/cksum -b -t --untagged --algo=md5 target/debug/coreutils
387c6f3f00118881383135c76cbb7129  target/debug/coreutils
$ ../gnu/src/cksum -t -b --untagged --algo=md5 target/debug/coreutils
387c6f3f00118881383135c76cbb7129 *target/debug/coreutils
```

Observe that `-b` and `-t` override each other, hence:

```suggestion
                .hide(true)
                .overrides_with(options::BINARY)
```

Reply:
ok, i missed this